spacy/tests/regression/test_issue2901.py

```diff
@@ -0,0 +1,13 @@
+import pytest
```

Comment:
Thanks for adding a test! I think the reason this fails on Python 2.7 is that it's missing a unicode declaration at the top of the file:

```python
# coding: utf8
from __future__ import unicode_literals
```

Reply:
Oops, I didn't think about that (since `mecab-python3` is only for Python 3). I've added support for Python 2 to https://github.com/kbulygin/spacy-dev because of this error.